test_scripts/API/Restructuring_OnResetTimeout/013_BC_DialNumber_with_BC_OnResetTimeout.lua

```diff
@@ -0,0 +1,65 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0189-Restructuring-OnResetTimeout.md
+-- User story: TBD
+-- Use case: TBD
+--
+-- Requirement summary: TBD
+--
+-- Description:
+-- In case:
+-- 1) DialNumber is requested
+-- 2) Some time after receiving DialNumber request on HMI is passed
+-- 3) HMI sends BC.OnResetTimeout(resetPeriod = 15000) to SDL
```

Comment:
(resetPeriod = 13000) is enough, soe please update, also the value resetPeriod

Reply:
@GetmanetsIrina Update